provider/bluecat/bluecat.go

```diff
@@ -802,7 +802,7 @@ func (c *GatewayClient) deleteCNAMERecord(name string) error {
 	url := c.Host + "/api/v1/configurations/" + c.DNSConfiguration +
 		"/views/" + c.View + "/" +
 		"cname_records/" + name + "/"
-	req, err := c.buildHTTPRequest("DELETE", url, nil)
+	req, _ := c.buildHTTPRequest("DELETE", url, nil)
```

Comment:
Instead of ignoring the error by adding an `_` I think we should add an error check like this.
```
if err != nil {
		return fmt.Errorf("error building http request: %s", name, err)
}
```